pkg/service/v0/service.go

```diff
@@ -102,35 +108,60 @@ func (o Ocs) GetUser(w http.ResponseWriter, r *http.Request) {
 // TODO middleware for the proxy
 func (o Ocs) GetSigningKey(w http.ResponseWriter, r *http.Request) {
 
-	// TODO move token marshaling to ocis-proxy
-	_, ok := user.ContextGetUser(r.Context())
+	// TODO the x access token is set but not unparsed
+	u, ok := user.ContextGetUser(r.Context())
 	if !ok {
-		//	render.Render(w, r, ErrRender(MetaBadRequest.StatusCode, "missing user in context"))
-		//	return
+		o.logger.Error().Msg("missing user in context")
+		render.Render(w, r, ErrRender(MetaBadRequest.StatusCode, "missing user in context"))
+		return
 	}
 	c := storepb.NewStoreService("com.owncloud.api.store", grpc.NewClient())
 	res, err := c.Read(r.Context(), &storepb.ReadRequest{
-		Key: "TODO replace with user from ctx",
 		Options: &storepb.ReadOptions{
-			Database: "ocs",
+			Database: "proxy",
 			Table:    "signing-keys",
-		}})
+		},
+		Key: u.GetUsername(), // TODO username or id?
```

Comment:
why not using the ocis account instead of the reva user?

Reply:
I think phoenix will send the username as the credential in signed requests on oc10 ... need to to get a phoenix with url signing enabled to test what happens on top of ocis.